Tasks/GithubRelease/operations/Action.ts

```diff
@@ -0,0 +1,111 @@
+import tl = require("vsts-task-lib/task");
+import path = require("path");
+import { WebResponse } from "./webClient";
+import { createRelease } from "./CreateRelease";
+import { uploadReleaseAsset } from "./UploadReleaseAsset";
+import { validateUploadAssets, getUploadAssets } from "./Utility";
+import { editRelease } from "./EditRelease";
+import { discardRelease } from "./DiscardRelease";
+import { deleteReleaseAsset } from "./DeleteReleaseAsset";
+
+export async function createReleaseAction(): Promise<void> {
+
+    try {
+        validateUploadAssets(); 
+        let response: WebResponse = await createRelease();
+
+        if (response.statusCode === 201) {
+            const uploadUrl: string = response.body["upload_url"];
+            await uploadAssets(uploadUrl, []);
+        }
+        else {            
+            throw new Error(tl.loc("ErrorCreateRelease") + "\n" + JSON.stringify(response));
```

Comment:
The response would be a huge string. I think you might not be interested to print that as a user-friendly error (will be printed in red color).  Better give a single line as a user-friendly error and print response as tl.debug